docker/ecr/Dockerfile

```diff
@@ -1 +1,6 @@
 FROM plugins/docker:latest
+
+RUN apk update && apk add ca-certificates
```

Comment:
This should be already done for plugins/docker so you can skip that

Reply:
I thought that was already installedbut it isn't.
Is it ok if I move `ca-certificates` to `docker/Dockerfile`? so other plugins have that package as well

Reply:
IMHO it have to be part of the base image, so it's plugins/docker